dnx_frontend/dfe_dnx_main.py

```diff
@@ -606,12 +606,13 @@ def update_session_tracker(username, user_role=None, remote_addr=None, *, action
 
     with ConfigurationManager('session_tracker', file_path='dnx_frontend') as session_tracker:
         stored_tracker = session_tracker.load_configuration()
-
+        
         if (action is CFG.ADD):
+            last_login = datetime.datetime.now()
```

Comment:
I found that putting datetime into a variable helps. Don't know why just what I have seen.